### PR TITLE
Fix(email): Send notifications on Comment

### PR DESCRIPTION
**Pull request by cirqlar**
#### This PR
1. Changes SMTP to Mailjet to prevent issues from google's security
2. Fixes comment notification sending

### CODE REVIEW

File: src/services/emailServices.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
-import { EMAIL_ADDR, EMAIL_PASSWORD } from '../config/constants';
+import { EMAIL_ADDR, EMAIL_PASSWORD, SENDER_ADDR } from '../config/constants';
```
Comment (shonubijerry) [inline src/services/emailServices.js:1]: Remove the `EMAIL_ADDR` constant as it's no longer used

**Review by shonubijerry — CHANGES_REQUESTED** [review]
Update the `.env.example` with this new environment credentials.

File: src/services/emailServices.js (hunk 1 above)
Comment (cirqlar) [reply src/services/emailServices.js:1]: The `EMAIL_ADDR` constant is still used.

**Comment by cirqlar** [comment]
.env.example has been updated

**Review by shonubijerry — APPROVED** [review]

**Pull request merged**